pulsar-client-tools/pom.xml

```diff
@@ -86,6 +86,40 @@
     </dependency>
 
     <!-- functions related dependencies (end) -->
+
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>pulsar-io-cassandra</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+
   </dependencies>
 
+  <build>
+    <plugins>
+      <!-- this task is used for copy all python files together to provide a instance directory
+           for running python instance -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>test-compile</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <tasks>
+                <echo>building python instance</echo>
```

Comment:
can you rename to something like "copy test sink package"?

Reply:
good catch